packages/playground/blueprints/public/blueprint-schema.json

```diff
@@ -698,6 +698,29 @@
 					},
 					"required": ["step", "themeZipFile"]
 				},
+				{
+					"type": "object",
+					"additionalProperties": false,
+					"properties": {
+						"progress": {
+							"type": "object",
+							"properties": {
+								"weight": {
+									"type": "number"
+								},
+								"caption": {
+									"type": "string"
+								}
+							},
+							"additionalProperties": false
+						},
+						"step": {
```

Comment:
I added this to have a [step inside compile](https://github.com/WordPress/wordpress-playground/pull/1789/files#diff-111618e10ba828ef4596cfba7f3a377a0c7cefc4b2432d96a13a9bd77f462987) which we can insert into the blueprint.

I could simply call the function from there, but it felt wrong because we don't execute any steps during compile.